site/content/en/docs/tasks/dev/develop-acc.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Develop an AdmissionCheck Controller"
+date: 2024-09-03
+weight: 9
+description: >
+    Develop an AdmissionCheck Controller (ACC).
+---
+
+An Admission Check Controller, sometime referred as **ACC**, is a component that manages the AdmissionChecks associated with it , by `spec.controllerName` and the workloads queued against ClusterQueues configured to use those AdmissionChecks.
+
+Check [Admission Check](/docs/concepts/admission_check/) section for the general description of the Admission Checks usage in Kueue.
+
+
+## Subcomponents
+
+The ACC can be built-in into Kueue or run in a different kubernetes controller manager and should implement reconcilers for AdmissionChecks and Workloads.
+
+### AdmissionCheck Reconciler
+
+Monitors the AdmissionChecks in the cluster and maintain the `Active` condition of those that are associated with it (by `spec.controllerName`). Optionally it can watch custom [parameters](/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-AdmissionCheckParametersReference) objects.
```

Comment:
```suggestion
Monitors the AdmissionChecks in the cluster and maintains the `Active` condition of those that are associated with it (by `spec.controllerName`).
Optionally, it can watch custom [parameters](/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-AdmissionCheckParametersReference) objects.
```